api-reference/_hidden/GridBaseColumn/isBand.md

```diff
@@ -5,54 +5,16 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies whether the column bands other columns or not.
+Specifies whether the column bands other columns.
 
 ---
-Unlike normal columns, band columns do not hold data. Instead, they collect two or more columns under one column header. In most cases, to set up this layout, you can declare the band column using a hierarchical structure. For example, the following code bands three columns under the *"Address"* header.
+Unlike normal columns, band columns do not hold data. Instead, they collect two or more columns under one column header. To create such a layout, do one of the following:
 
----
-##### jQuery
+* [Nest column configurations inside each other]({basewidgetpath}/Configuration/columns/columns/).
```

Comment:
Can you clarify what you mean here? (I'm not quite sure how you nest a column configuration.) For example, do you mean:

`Nest one column within another.`

Reply:
I mean such a structure that nests `columns` options.
e.g.:
columns: [{
    caption: "Nominal GDP",
    columns: [{
        caption: "Total, mln $",
        dataField: "GDP_Total",
        format: "fixedPoint",
        sortOrder: "desc"
    }, {
        caption: "By Sector",
        columns: [{
            caption: "Agriculture",
            dataField: "GDP_Agriculture",
            width: 95,
            format: {
                type: "percent",
                precision: 1
            }
        }, {
            caption: "Industry",
            dataField: "GDP_Industry",
            width: 80,
            format: {
                type: "percent",
                precision: 1
            }
        }, {
            caption: "Services",
            dataField: "GDP_Services",
            width: 85,
            format: {
                type: "percent",
                precision: 1
            }
        }]
    }]
}]

Reply:
Thank you. Can we say the following?

```suggestion
* [Create nested column configurations]({basewidgetpath}/Configuration/columns/columns/).
```

Reply:
Yes, it's better.

Reply:
Thank you. Please apply the suggestion.